packages/runtime/container-runtime/src/test/summarizerNode.spec.ts

```diff
@@ -293,7 +293,7 @@ describe("Runtime", () => {
 					// Validate summary fails by calling completeSummary.
 					assert.throws(
 						() => rootNode.completeSummary("test-handle", true /* validateSummary */),
-						(error) => {
+						(error: any) => {
```

Comment:
@agarwal-navin Is there a more correct type to use here and elsewhere in this file?

Reply:
The code thrown an `ILoggingError`.

Reply:
I wasn't able to use ILoggingError because it doesn't have the `id` property. I left these as any to get this in - please follow up if you want to add more explicit typing.